Facebook.Android/source/facebook-android-sdk/facebook-android-sdk.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
```

Comment:
According to the dependencies I saw on maven central, there might be some changes.

Was made sure, that no adaption to the dependency graph between the C# projects is needed?

Reply:
Sorry, I didn't check on this, as the projects built in the same order as before without issues.